warehouse/query-core/src/main/java/datawave/query/index/lookup/ConcurrentScannerInitializer.java

```diff
@@ -35,6 +35,10 @@ public ConcurrentScannerInitializer(BaseIndexStream stream) {
     public BaseIndexStream call() throws Exception {
         if (stream.context() == StreamContext.INITIALIZED) {
             if (stream.hasNext()) {
+                if (stream instanceof ScannerStream) {
```

Comment:
a comment as to what is going on here please.

Reply:
Added in https://github.com/NationalSecurityAgency/datawave/pull/1803/commits/5656aa2a8c6ec0f910ff6e83b3b55f043c798462